sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs

```diff
@@ -73,18 +74,19 @@ public TimeSeriesInsightsClient(string environmentFqdn, TokenCredential credenti
         /// </remarks>
         public TimeSeriesInsightsClient(string environmentFqdn, TokenCredential credential, TimeSeriesInsightsClientOptions options)
         {
+            Argument.AssertNotNullOrEmpty(environmentFqdn, nameof(environmentFqdn));
+            Argument.AssertNotNull(credential, nameof(environmentFqdn));
             Argument.AssertNotNull(options, nameof(options));
 
+            _clientSessionId = new Guid().ToString();
```

Comment:
Is this supposed to be a Guid with all 0s? if you intend to create a random guid, you have to use `Guid.NewGuid()`

Reply:
good catch.

Reply:
Azad, I decided to leave this as is as we plan to address it in a separate PR. Leaving it for now as changing it is messing with the recording of tests since every run is a new Guid.